### PR TITLE
CY-1463: Fix major issue for port/networks ordering attached to serve…

### PR DESCRIPTION
**Pull request by mabuaisha**
…r instance

### CODE REVIEW

File: openstack_plugin/resources/compute/server.py [hunk 1]
```diff
@@ -510,28 +516,88 @@ def _get_port_networks(client_config, port_ids):
...
+    if port_nic:
+        for item in nics_from_rels:
+            data = tuple(sorted(item.items()))
+            if data[0][1] in port_nic.keys():
+                data = data + (('port', port_nic[data[0][1]]),)
+            elif data[0][1] in port_nic.values():
+                data = (('uuid', inverted_port_nic[data[0][1]]),) + data
+            if data not in unique_set:
+                unique_set.add(data)
+                ordered_list.append(dict(data))
```
Comment (EarthmanT) [inline openstack_plugin/resources/compute/server.py]: Can we change the name of "data" to something more readable? :)

File: openstack_plugin/resources/compute/server.py [hunk 2]
```diff
@@ -510,28 +516,88 @@ def _get_port_networks(client_config, port_ids):
...
+    to API
     """
-    networks = server_config.get('networks')
-    if networks:
-        networks = remove_duplicates_items(networks)
-        server_config['networks'] = networks
+    for node_nic in nics_from_node:
+        node_port_id = node_nic.get('port')
+        node_nic_id = node_nic.get('uuid')
+        for rel_nic in nics_from_rels:
```
Comment (EarthmanT) [inline openstack_plugin/resources/compute/server.py]: Can you add some comment explaining the logic?

Comment (mabuaisha) [reply openstack_plugin/resources/compute/server.py]: Sure

File: openstack_plugin/resources/compute/server.py (hunk 1 above)
Comment (mabuaisha) [reply openstack_plugin/resources/compute/server.py]: Yup

**Pull request merged**